.github/workflows/stale.yml

```diff
@@ -28,10 +28,6 @@ jobs:
           days-before-close: 5
           operations-per-run: 300
           remove-stale-when-updated: true
-          stale-issue-label: ${{ env.STALE_LABEL }}
-          exempt-issue-labels: ${{ env.EXEMPT_LABELS }}
-          stale-issue-message: ${{ env.STALE_MESSAGE }}
-          close-issue-message: ${{ env.CLOSE_MESSAGE }}
           stale-pr-label: ${{ env.STALE_LABEL }}
```

Comment:
```diff
# stale.yml
- env:
-   STALE_LABEL: stale
-   EXEMPT_LABELS: keep
-   STALE_MESSAGE: |
-     This has been automatically marked as stale because it hasn't had any recent activity. 
-     It will be closed in 5 days if no further activity occurs.
-   CLOSE_MESSAGE: |
-     Closing this because it has been inactive for 5 days since being marked as stale.
```

PR만 적용한다면 env를 사용하지 않아도 될 것 같습니다!

Reply:
@jongwooo 님, stale 라벨 달린 이슈는 닫으려구요. 단, stale 라벨을 자동 추가하지는 않고자 합니다.

Reply:
@soimkim , `days-before-issue-stale: -1`로 override, `stale-issue-label`, `close-issue-message` 설정은 다시 추가해주시고 나머지는 기존의 설정 그대로 두어도 괜찮을 것 같습니다. 

```diff
# stale.yml
  days-before-stale: 30
  days-before-close: 5
+ days-before-issue-stale: -1
  ...
+ stale-issue-label: ${{ env.STALE_LABEL }}
- exempt-issue-labels: ${{ env.EXEMPT_LABELS }}
- stale-issue-message: ${{ env.STALE_MESSAGE }}
+ close-issue-message: ${{ env.CLOSE_MESSAGE }}
```

Reply:
앗 맞네요!! 다시 살려야하네요

Reply:
@jongwooo 님, 땡큐!!! 종우님 커밋이네용!! 👍